src/sentry/user_feedback/blueprints/api.md

```diff
@@ -0,0 +1,150 @@
+
+Retrieve a collection of user-feedback items.
+
+**Attributes**
+
+| Column        | Type             | Description                                         |
+| ------------- | ---------------- | --------------------------------------------------- |
+| email         | string           | The contact email of the user writing feedback.     |
+| error_ids     | array[string]    | A list of error_ids encountered during the session. |
+| feedback_text | string           | The message written by the user providing feedback. |
+| id            | string           |                                                     |
+| replay_id     | optional[string] | The id of a running replay.                         |
+| timestamp     | string           | ISO-8061 formatted datetime.                        |
+| trace_ids     | array[string]    | A list of trace_ids.                                |
+| url           | string           | The page the user-feedback was triggered on.        |
+
+- Response 200
+
+  ```json
+  {
+    "data": [
+      {
+        "email": "colton.allen@sentry.io",
+        "error_ids": ["4f0e18332d11431c9b97d924edecb76e"],
+        "feedback_text": "I really like this user-feedback feature!",
+        "id": "1ffe0775ac0f4417aed9de36d9f6f8dc",
+        "replay_id": "ec3b4dc8b79f417596f7a1aa4fcca5d2",
+        "timestamp": "2023-08-31T14:10:34.954048",
+        "trace_ids": ["fbb42a39a61e4aa6969b4fecbfbbc114"],
+        "url": "https://docs.sentry.io/platforms/javascript/"
+      }
+    ]
+  }
+  ```
+
+### Create User Feedback [POST]
+
+Create a new user-feedback item.
+
+- Request
+
+  ```json
+  {
+    "data": {
+      "email": "colton.allen@sentry.io",
+      "error_ids": ["4f0e18332d11431c9b97d924edecb76e"],
+      "feedback_text": "I really like this user-feedback feature!",
+      "id": "1ffe0775ac0f4417aed9de36d9f6f8dc",
+      "replay_id": "ec3b4dc8b79f417596f7a1aa4fcca5d2",
+      "timestamp": "2023-08-31T14:10:34.954048",
+      "trace_ids": ["fbb42a39a61e4aa6969b4fecbfbbc114"],
+      "url": "https://docs.sentry.io/platforms/javascript/"
+    }
+  }
+  ```
+
+## User Feedback [/projects/<organization_slug>/<project_slug>/user-feedback/<user_feedback_id>/]
+
+- Model
+
+  - Body
+
+    ```json
+    {
+      "data": {
+        "email": "colton.allen@sentry.io",
+        "error_ids": ["4f0e18332d11431c9b97d924edecb76e"],
+        "feedback_text": "I really like this user-feedback feature!",
+        "id": "1ffe0775ac0f4417aed9de36d9f6f8dc",
+        "replay_id": "ec3b4dc8b79f417596f7a1aa4fcca5d2",
+        "timestamp": "2023-08-31T14:10:34.954048",
+        "trace_ids": ["fbb42a39a61e4aa6969b4fecbfbbc114"],
+        "url": "https://docs.sentry.io/platforms/javascript/"
+      }
+    }
+    ```
+
+### Fetch User Feedback [GET]
+
+Retrieve a single user-feedback item.
+
+- Response 200
+
+  [UserFeedback]
+
+### Update User Feedback [PATCH]
+
+Partially update a user-feedback item.
+
+- Request
+
+  ```json
+  {
+    "data": {}
```

Comment:
It's not clear to me if the user-feedback item should have an assigned_to field where you can assign a user in your organization or if there should be a secondary m2m table which accepts user_feedback_id, user_id.